client/src/Component/MainTableStyle.css

```diff
@@ -1,8 +1,23 @@
-.styled-table {
-    margin: 1rem 0;
-    font-size: 0.9em;
+.align-bikes-needed {
+    text-align: center;
 }
 
-.styled-table tbody tr:last-of-type{
-    border-bottom: 3px solid #000000;  
-}
\ No newline at end of file
+.btn-on-hover:hover {
```

Comment:
would it maybe be neater for these two buttons to give them IDs and style them that way? i was thinking something like `id="csv-btn"` and `id="sms-btn"`

Reply:
We typically use classes rather than IDs to style elements because:

1. It's semantic—the `class` attribute was created for css styling
2. Classes can be reused across many elements on the page, whereas an element's ID must be unique on the page so you can't reuse them

But yes, it's common to name classnames based on what the element is, so classnames like `csv-btn` and `sms-btn` sound fine

Reply:
ahhhh thank you!